schema.sql

```diff
@@ -9,5 +9,20 @@ CREATE TABLE animals (
     weight_kg decimal
 );
 
- ALTER TABLE animals 
- ADD column species varchar(100);
\ No newline at end of file
+ALTER TABLE animals 
+ADD column species varchar(100);
+
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    full_name varchar(100),
+    age integer
+);
+
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name varchar(100) NOT NULL
+);
+
+ALTER TABLE animals DROP column species;
+ALTER TABLE animals ADD column species_id int REFERENCES species(id);
+ALTER TABLE animals ADD column owner_id int REFERENCES owners(id);
```

Comment:
- [ ] As it is [required](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md) for this project, kindly add the following files and their contents. 

    - `data.sql file`
       Insert the following data into the owners table:
  - Sam Smith 34 years old.
  - Jennifer Orwell 19 years old.
  - Bob 45 years old.
  - Melody Pond 77 years old.
  - Dean Winchester 14 years old.
  - Jodie Whittaker 38 years old.
  - Insert the following data into the species table:
Pokemon
Digimon
   - Modify your inserted animals so it includes the species_id value:
  - If the name ends in "mon" it will be Digimon
  - All other animals are Pokemon
  - Modify your inserted animals to include owner information (owner_id):
  - Sam Smith owns Agumon.
  - Jennifer Orwell owns Gabumon and Pikachu.
  - Bob owns Devimon and Plantmon.
  - Melody Pond owns Charmander, Squirtle, and Blossom.
  - Dean Winchester owns Angemon and Boarmon.

- `queries.sql file` 
- Write queries (using JOIN) to answer the following questions:
  - What animals belong to Melody Pond?
  - List of all animals that are pokemon (their type is Pokemon).
  - List all owners and their animals, remember to include those that don't own any animal.
  - How many animals are there per species?
  - List all Digimon owned by Jennifer Orwell.
  - List all animals owned by Dean Winchester that haven't tried to escape.
  - Who owns the most animals?